build.gradle

```diff
@@ -0,0 +1,68 @@
+plugins {
+    id 'java'
+    id 'application'
+    id 'org.openjfx.javafxplugin' version '0.0.10'
+    id 'org.beryx.jlink' version '2.24.1'
+}
+
+group 'io.rpg'
+version '1.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+ext {
+    junitVersion = '5.8.1'
+}
+
+sourceCompatibility = '17'
+targetCompatibility = '17'
+
+tasks.withType(JavaCompile) {
+    options.encoding = 'UTF-8'
+}
+
+application {
+    mainModule = 'io.rpg.rpg'
+    mainClass = 'io.rpg.rpg.HelloApplication'
+}
+
+javafx {
+    version = '15.0.1'
+    modules = ['javafx.controls', 'javafx.fxml', 'javafx.web']
+}
+
+dependencies {
+    implementation('org.controlsfx:controlsfx:11.1.1')
+    implementation('com.dlsc.formsfx:formsfx-core:11.5.0') {
+        exclude(group: 'org.openjfx')
+    }
+//    implementation('net.synedra:validatorfx:0.1.13') {
+//        exclude(group: 'org.openjfx')
+//    }
+    implementation('org.kordamp.ikonli:ikonli-javafx:12.3.0')
+    implementation('org.kordamp.bootstrapfx:bootstrapfx-core:0.4.0')
+//    implementation('eu.hansolo:tilesfx:17.1.5') {
+//        exclude(group: 'org.openjfx')
+//    }
```

Comment:
The project did not build successfully with these two dependencies. Had to cut them out. Don't think we will need them. Even if so, we can figure it out later on.